knowledge_storm/lm.py

```diff
@@ -232,6 +232,113 @@ def get_usage_and_reset(self):
 
         return usage
 
+class GroqOpenAIModel(dspy.OpenAI):
```

Comment:
This class name is unintuitive given that this is not for running OpenAI model (e.g., GPT family). Could you change a name?

Reply:
Agreed. Changed the class name to GroqModel in line with DeepSeekModel, which also otherwise uses dspy.OpenAI.